userguide/tutorials/what_is_kill_bill.adoc

```diff
@@ -34,21 +38,30 @@ So, in the diagram above, we can see that:
 
 * Customers interact with your website, which then issues REST API calls to Kill Bill. **You are responsible for designing and building that piece**.
 * Kill Bill, with its database, **manages all the state associated with the subscriptions, invoices, payments, etc.**
-* KAUI is the **Back-office UI for your teams**, and it includes role based access control to provide different priviledges for different teams/users:
+* Kaui is the **Back-office UI for your teams**, and it includes role based access control to provide different priviledges for different teams/users:
   ** Your customer support team will use to access/modify consumer accounts data
   ** Your product team  will use to configure catalog information, billing rules, ...
   ** You finance/accounting team will access it for different types of reports
-  ** Your engineering team will access it for running quick dev scenarios
+  ** Your engineering team will access it for running quick scenarios for development and testing purposes
 
 
 === Deployment
 
-Kill Bill is packaged as a WAR and is therefore executed within a web container (typically Tomcat). While you can deploy Kill Bill directly as a WAR inside a container, we also provide easier deployment options:
+Kill Bill is packaged as a self-contained WAR and is therefore executed within a web container (typically Tomcat). While you can deploy Kill Bill directly as a WAR inside a container, we also provide additional deployment options, such as Docker containers or _1-click button_ AWS AMIs.
+
+Here is a typical Kill Bill deployment architecture:
+
+[caption=""]
+//.Kill Bill: Zoomed-in Picture
+image::../assets/what_is_killbill/KillBill_Mid_level.png[format=png,align=center,title-align=center]
```

Comment:
This is the **physical** view of the deployment, while the first diagram, the 'high level' is the **logical** view

Reply:
I would change the olive-green color of the JDBC line to something brighter. Right now it's color is too close to black.

Reply:
I've renamed the caption to mention *physical* view.

I've also changed the colour to something brighter.